server/constants/errors.js

```diff
@@ -33,6 +33,21 @@ export class DuplicateEmailAddressError extends Error {
   }
 }
 
+// Do i need to update every single code in order to add a new one?! :scream:
+export class EmailNotFoundError extends Error {
```

Comment:
Could we move these to the end of the file to make sure code numbers are in ASC order?